github-clone.el

```diff
@@ -180,7 +186,7 @@ DIRECTORY.  Then it prompts to fork the repository and add a
 remote named after the github username to the fork."
   (interactive
    (list (read-from-minibuffer "Url or User/Repo: ")
-         (read-directory-name "Directory: " nil default-directory t)))
+         (read-directory-name "Directory: " github-clone-default-directory nil t)))
```

Comment:
> Alternatively perhaps the right way to do this is to make the default value for github-clone-default-directory to be an alias of the value of default-directory, so it only overrides if specified.

I would vote for that.  I always clone to one of two directories, and primarily one of them, and never `~`, so I'd like to not have to choose the directory when I'm using my standard one.  :)